Modules/IO/NIFTI/test/itkNiftiImageIOTest14.cxx

```diff
@@ -162,8 +159,6 @@ itkNiftiImageIOTest14(int argc, char * argv[])
     throw;
   }
 
-  itk::IOTestHelper::Remove(output_test_fn.c_str());
-
```

Comment:
@thewtex Are you sure that the created output file should not be removed afterwards?

Reply:
It is better to keep it for inspection.